tests/page-objects/rerences-types.ts

```diff
@@ -0,0 +1,149 @@
+export type Names = {
+  lastName: string;
+  firstName: string;
+}[];
+
+export type CommonReferenceProperties = {
+  type: string;
+  names: Names;
+  doi?: string;
+}
+
+export type JournalReference = CommonReferenceProperties & {
```

Comment:
Interesting, I learnt today that you can't extend types using `extends`! Guess I just use interfaces too much. Anyway, good use of pseudo inheritance!